Tarball missing CMakeLists.txt file

When trying to build 1.7.4 from tarball using CMake, I get the following error:

```
CMake Error at CMakeLists.txt:690 (add_subdirectory):
  The source directory

    /tmp/nix-build-harfbuzz-1.7.4.drv-0/harfbuzz-1.7.4/test

  does not contain a CMakeLists.txt file.


-- Configuring incomplete, errors occurred!
```

Inspecting the downloaded [tarball](https://www.freedesktop.org/software/harfbuzz/release/harfbuzz-1.7.4.tar.bz2) confirms the file is indeed missing.

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -59,7 +59,8 @@ jobs:
       - checkout
       - run: apt update && apt install -y pkg-config ragel gtk-doc-tools libfreetype6-dev libglib2.0-dev libcairo2-dev libicu-dev libgraphite2-dev
       - run: ./autogen.sh && make distdir
-      - run: cd harfbuzz-* && ./configure && make && make check && make install
+      - run: cd harfbuzz-* && ./configure && make && make check && make install && make clean
+      - run: cd harfbuzz-* && cmake -DHB_HAVE_FREETYPE=ON -DHB_HAVE_GLIB=ON -DHB_BUILD_UTILS=ON -Bbuild -H. -GNinja && ninja -Cbuild && ninja -Cbuild test && ninja -Cbuild install
 
   psvita:
     docker:

--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -1,6 +1,12 @@
 # Process this file with automake to produce Makefile.in
 
+NULL =
+EXTRA_DIST =
 SUBDIRS = api shaping fuzzing
+
+EXTRA_DIST += \
+	CMakeLists.txt \
+	$(NULL)
 
 # Convenience targets:
 lib:

--- a/test/api/Makefile.am
+++ b/test/api/Makefile.am
@@ -14,6 +14,7 @@ if HAVE_GLIB
 AM_CPPFLAGS = -DSRCDIR="\"$(srcdir)\"" -I$(top_srcdir)/src/ -I$(top_builddir)/src/ $(GLIB_CFLAGS)
 LDADD = $(top_builddir)/src/libharfbuzz.la $(GLIB_LIBS)
 
+EXTRA_DIST += CMakeLists.txt
 EXTRA_DIST += hb-test.h
 
 check_PROGRAMS = $(TEST_PROGS)

